README.md

```diff
@@ -234,29 +249,37 @@ And what is this ownership worth?
   or likely future ownership.
   In practice, this can mean, stock, stock options, and RSUs, which we’ll divide and conquer
   later.
-  - 👆 The word “equity” has
+  - ☝️ The word “equity” has
     [several technical meanings](https://www.investopedia.com/terms/e/equity.asp) in accounting
     and other financial contexts, but when we talk about equity compensation, it is meant more
     informally.
-- **Why give equity compensation?**
-  The purpose of equity compensation is twofold.
-  - To attract the best talent:
-    When a company already has or can be predicted to have significant financial success, the
-    most talented people are incentivized to work for the company by the prospect of their
-    equity turning into large amounts of cash in the future.
-  - To align individuals’ incentives with the interests of the company:
-    Even companies that can afford to pay lots of cash may prefer to give employees equity, so
-    that employees work to increase the *future* value of the company.
-    At its best, this philosophy fosters an environment of teamwork and a “rising tides lift
-    all boats” mentality.
-    As we’ll talk more about during our chapter on [negotiations](#negotiations-and-offers), the
-    amount of equity you’re offered usually reflects both your past or current contribution to
-    the company and your commitment to the company in the future.
+
+### Why give equity compensation?
+
+The purpose of equity compensation is threefold:
+
+- To attract the best talent:
+  When a company already has or can be predicted to have significant financial success, the
+  most talented people are incentivized to work for the company by the prospect of their
+  equity turning into large amounts of cash in the future.
+- To align individuals’ incentives with the interests of the company:
+  Even companies that can afford to pay lots of cash may prefer to give employees equity, so
+  that employees work to increase the *future* value of the company.
+  At its best, this philosophy fosters an environment of teamwork and a “rising tides lift
+  all boats” mentality.
+  As we’ll [discuss later](#offers-and-negotiations), the amount of equity you’re offered
+  usually reflects both your past or current contribution to the company and your commitment
+  to the company in the future.
+- To reduce the company’s spending on cash compensation:
```

Comment:
Simon (Dummies) refers to 'cash diversion.' We may want to 🌪 this and talk a bit about the potential for abuse of this tactic? And controversy of skyrocketing equity-based pay for CEOs (data and interesting figures in this 2013 study: https://www.haygroup.com/downloads/us/wsj%20hay%20group%202013%20ceo%20compensation%20study.pdf. That year, 37% CEO pay is in cash, 54% in stock and options. Equity pay increased after 2008 financial crises, connecting us back to diverted cash.

Reply:
If we open an evaluation of this it probably requires more than that single link on CEO compensation, which is just one aspect. I'll track this issue separately.